File: Day 17/p3.py
# 🔹 3.
# Read students.json and print each student's name and average grade.



import json


with open("Day 17/students.json") as f:
    data = json.load(f) 


for student in data:
    name = student["name"]
    grades = student["grades"]
    average = sum(grades) / len(grades)
    print(f"{name}'s average grade is: {average:.2f}")

File: Day 39/p3.js
// 🔹 3. Student Object Summary
// Create an object student with name, age, and grades (array). Write a method to print the average grade.



const student = {
  name: "Anshuman",
  age: 19,
  grades: [90, 85, 92],
  getAverage: function () {
    let total = 0;
    for (let grade of this.grades) {
      total += grade;
    }
    return total / this.grades.length;
  }
};

console.log(`Average Grade: ${student.getAverage()}`);
// node "Day 39/p3.js"